monsteer/qt/qTypes.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2011-2015, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+ *
+ * This file is part of Monsteer <https://github.com/BlueBrain/Monsteer>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _GuiTypes_h_
+#define _GuiTypes_h_
+
+#include <QAbstractTableModel>
```

Comment:
Mmm... it is used, but indirectly in several other files that include 
<monsteer/qt/qTypes.h>. I decided to put it here to have it only once.

On 10/20/2015 04:46 PM, Raphael Dumusc wrote:
>
> In monsteer/qt/qTypes.h 
> <https://github.com/BlueBrain/Monsteer/pull/6#discussion_r42504866>:
>
> > + * by the Free Software Foundation.
> > + *
> > + * This library is distributed in the hope that it will be useful, but WITHOUT
> > + * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
> > + * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
> > + * details.
> > + *
> > + * You should have received a copy of the GNU Lesser General Public License
> > + * along with this library; if not, write to the Free Software Foundation, Inc.,
> > + * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
> > + */
> > +
> > +#ifndef _GuiTypes_h_
> > +#define _GuiTypes_h_
> > +
> > +#include <QAbstractTableModel>
>
> unused include..?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BlueBrain/Monsteer/pull/6/files#r42504866>.
>